### PR TITLE
Fix workspace integration test suite and reduce log verbosity

### PR DESCRIPTION
**Pull request by aledbf**
## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-integration-tests=workspace
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by utam0k — APPROVED** [review]
Thanks @aledbf

File: test/tests/components/ws-daemon/cpu_burst_test.go [hunk 1]
```diff
@@ -26,8 +26,8 @@ import (
 type DaemonConfig struct {
 	CpuLimitConfig struct {
 		Enabled    bool  `json:"enabled"`
-		Limit      int64 `json:"limit,string"`
-		BurstLimit int64 `json:"burstLimit,string"`
+		Limit      int64 `json:"limit"`
+		BurstLimit int64 `json:"burstLimit"`
```
[the remark this answers is not in the document]
Comment (jenting) [reply test/tests/components/ws-daemon/cpu_burst_test.go:30]: PR to revert this change #14875.

[the remark this answers is not in the document]
Comment (aledbf) [reply test/tests/components/ws-daemon/cpu_burst_test.go:30]: https://gitpod.slack.com/archives/C03E52788SU/p1668643970940509?thread_ts=1668618551.920679&cid=C03E52788SU

[the remark this answers is not in the document]
Comment (jenting) [reply test/tests/components/ws-daemon/cpu_burst_test.go:30]: Interesting. So we have two data types: string and integer.

**Pull request merged**